insights/parsers/tests/test_cpu_vulns.py

```diff
@@ -0,0 +1,171 @@
+from insights.parsers import SkipException
+from insights.tests import context_wrap
+from insights.parsers import cpu_vulns
+from insights.parsers.cpu_vulns import CpuVulns
+import pytest
+import doctest
+
+INPUT_MELTDOWN = """
+Mitigation: PTI
+""".strip()
+
+INPUT_SPECTRE_V1 = """
+Mitigation: Load fences
+""".strip()
+
+INPUT_SPECTRE_V2_RHEL_7_1 = """
+Mitigation: Full generic retpoline, IBPB: conditional, IBRS_FW, STIBP: conditional, RSB filling
+""".strip()
+
+INPUT_SPECTRE_V2_RHEL_7 = """
+Vulnerable: Retpoline without IBPB
+""".strip()
+
+INPUT_SPECTRE_V2_RHEL_6 = """
+Mitigation: IBRS (kernel)
+""".strip()
+
+INPUT_SPEC_STORE_BYPASS = """
+Mitigation: Speculative Store Bypass disabled
+""".strip()
+
+INPUT_SPEC_STORE_BYPASS_2 = """
+Not affected
+""".strip()
+
+INPUT_SPEC_STORE_BYPASS_3 = """
+Vulnerable
+""".strip()
+
+INPUT_SMT = """
+Mitigation: PTE Inversion; VMX: conditional cache flushes, SMT vulnerable
+""".strip()
+
+# This could occur if you've not updated to the latest intel-microcode package or if Intel has not released new microcode for your processor. You'll see the following in this situation:
+INPUT_MDS = """
+Vulnerable: Clear CPU buffers attempted, no microcode; SMT vulnerable
+""".strip()
+
+# Processors that aren't vulnerable to MDS will report the following:
+INPUT_MDS_2 = """
+Not affected
+""".strip()
+
+# Processors that have Hyper-Threading support enabled will indicate that SMT is vulnerable:
+INPUT_MDS_3 = """
+Mitigation: Clear CPU buffers; SMT vulnerable
+""".strip()
+
+# The file will contain the following contents for processors that do not support Intel Hyper-Threading or where Hyper-Threading has been disabled:
+INPUT_MDS_4 = """
+Mitigation: Clear CPU buffers; SMT disabled
+""".strip()
+
+# The kernel is unable to reliably determine whether Hyper-Threading is enabled when running in a virtual environment. Updated host kernel packages, updated host qemu packages with proper configuration to pass through the host CPU type to the guest, and updated guest kernel packages will show the following status inside of the virtual environment:
+INPUT_MDS_5 = """
+Mitigation: Clear CPU buffers; SMT Host state unknown
+""".strip()
+
+
+# CpuVulnsMeltdown
+def test_cpu_vulns_meltdown():
+    spectre = CpuVulns(context_wrap(INPUT_MELTDOWN, path='meltdown'))
```

Comment:
Should be real path we expect: `path='/sys/devices/system/cpu/vulnerabilities/meltdown'`

This applies for all tests.

Reply:
Yes, it sounds make sense. Let me modify it to real paths.